### ISSUE TITLE
Field content disappears when a Vizy block with a table is not saved

### ISSUE THREAD

**Issue by scandella** [verbb/vizy#237]
### Describe the bug

When adding a table from the editor toolbar but not saving the entry and then reopening the entry, all the Vizy content is gone.
This happens even if a table is already there in a previously saved entry: exiting the entry without saving makes all the content disappear when reopening the same entry.
You have to cancel the modification to get the content back and sometimes you're stuck in restoring the previous entry loop that never restore the full content.

### Steps to reproduce

1. Insert some content in a Vizy block
2. Insert a table from the toolbar editor anywhere
3. Do not save the entry
4. Return to the main entry list
5. Click on the same entry 
6. The Vizy block is empty

### Craft CMS version

Craft Pro 4.4.15

### Plugin version

2.1.6

### Multi-site?

No

### Additional context

To summarize, any Vizy block with a table in it returns empty when not saving the entry.
Maybe it happens with some other element. Seem to be constant with a table.

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/vizy:"dev-craft-4 as 2.1.6"`

**Comment by scandella** [comment]
Thanks so much for the quick fix. It worked!
Very good support.

**Comment by engram-design** [comment]
Fixed in [2.1.7](https://github.com/verbb/vizy/releases/tag/2.1.7)
